src/pages/home/report/ReportActionsView.tsx

```diff
@@ -132,7 +132,12 @@ function ReportActionsView({
         setCurrentReportActionID('');
     }, [route]);
 
+    // Change the list ID only for comment linking to get the positioning right
     const listID = useMemo(() => {
+        if (!reportActionID) {
+            // Keep the old list ID since we're not in the Comment Linking flow
+            return listOldID;
+        }
```

Comment:
We missed the case where we switch from a report with comment linking to a report without comment linking, in this case we shouldn't use the same list id

- reportActionID: 123 -> reportActionID: 555 => Use different id
- **reportActionID: 123 -> reportActionID: null => Use different id**
- reportActionID: null -> reportActionID: 123 => Use different id
- reportActionID: null -> reportActionID: null => Use same id

(Coming from https://github.com/Expensify/App/issues/45004)